src/modules/85-iacm/pages/pipeline-deployment-list/__tests__/IACMPipelineDeploymentList.test.tsx

```diff
@@ -0,0 +1,198 @@
+        metaData: null,
+        data: {}
+      }
+    })
+  }))
+}))
+
+jest.mock('services/pipeline-rq', () => ({
+  useGetPipelineSummaryQuery: jest.fn(() => PipelineDetailsMockResponse)
+}))
+
+const getListOfBranchesWithStatus = jest.fn(() => Promise.resolve(branchStatusMock))
+const getListGitSync = jest.fn(() => Promise.resolve(gitConfigs))
+
+jest.mock('services/cd-ng', () => ({
+  useGetServiceDefinitionTypes: jest
+    .fn()
+    .mockImplementation(() => ({ loading: false, data: deploymentTypes, refetch: jest.fn() })),
+  useGetServiceListForProject: jest
+    .fn()
+    .mockImplementation(() => ({ loading: false, data: services, refetch: jest.fn() })),
+  useGetEnvironmentListForProject: jest
+    .fn()
+    .mockImplementation(() => ({ loading: false, data: environments, refetch: jest.fn() })),
+  useGetListOfBranchesWithStatus: jest.fn().mockImplementation(() => {
+    return { data: branchStatusMock, refetch: getListOfBranchesWithStatus, loading: false }
+  }),
+  useCreatePR: jest.fn(() => noop),
+  useCreatePRV2: jest.fn(() => noop),
+  useGetFileContent: jest.fn(() => noop),
+  useGetFileByBranch: jest.fn().mockImplementation(() => ({ refetch: jest.fn() })),
+  useGetGlobalFreezeWithBannerDetails: jest.fn().mockReturnValue({ data: null, loading: false }),
+  useListGitSync: jest.fn().mockImplementation(() => {
+    return { data: gitConfigs, refetch: getListGitSync }
+  })
+}))
+
+jest.mock('services/template-ng', () => ({
+  useGetYamlWithTemplateRefsResolved: jest.fn(() => ({}))
+}))
+
+jest.mock('services/cd-ng-rq', () => ({
+  useGetSourceCodeManagersQuery: jest.fn().mockImplementation(() => {
+    return { data: sourceCodeManagers, refetch: jest.fn() }
+  })
+}))
+
+const renderExecutionPage = (): RenderResult =>
+  render(
+    <TestWrapper
+      path={routes.toPipelineDeploymentList({ ...accountPathProps, ...pipelinePathProps, ...pipelineModuleParams })}
+      pathParams={{
+        accountId: 'accountId',
+        orgIdentifier: 'orgIdentifier',
+        projectIdentifier: 'projectIdentifier',
+        pipelineIdentifier: 'pipelineIdentifier',
+        module: 'iacm'
+      }}
+      defaultAppStoreValues={defaultAppStoreValues}
+      queryParams={{ listview: true }}
+    >
+      <IACMPipelineDeploymentList />
+    </TestWrapper>
+  )
+
+describe('IACMPipelineDeploymentList', () => {
+  beforeAll(() => {
+    jest.spyOn(global.Date, 'now').mockReturnValue(1603645966706)
+  })
+  afterAll(() => {
+    jest.spyOn(global.Date, 'now').mockReset()
+  })
+
+  test('should render pipelines', async () => {
+    renderExecutionPage()
+    const pipeline = await screen.findByRole('link', {
+      name: 'Sonar PR CI webhook PR'
+    })
+    expect(pipeline).toHaveAttribute(
+      'href',
+      '/account/accountId/iacm/orgs/orgIdentifier/projects/projectIdentifier/pipelines/pipelineIdentifier/executions/fP7dMHs9SP6375wqLc9bkA/pipeline'
+    )
+  })
+
+  test('should be able to show any pipeline`s executions', async () => {
```

Comment:
Backticks probably aren't a good substitute for an apostrophe.
```suggestion
  test("should be able to show any pipeline's executions", async () => {
```